python-sdk/tests/sql/operators/test_drop_table.py

```diff
@@ -53,7 +53,7 @@ def test_drop_table_with_table_metadata(database_table_fixture, sample_dag):
     def do_nothing(df: pandas.DataFrame):
         return df
 
-    @task
```

Comment:
The Astro SDK wouldn't connect to standard Task Flow API tasks is the biggest downside of this approach, in my opinion.

I'd love to see how this approach compares to the Custom Backend solution. Would we have any other alternatives?

Reply:
@tatiana We can make the custom backend for taskflow parity, but that would need to be a separate PR as this PR is already quite large and changes a lot. I think we should turn `astro.dataframe` into `astro.python` so people understand it's not just for dataframes.